dojo/unittests/tools/test_jfrog_xray_unified_parser.py

```diff
@@ -283,18 +288,19 @@ def test_parse_file_with_very_many_vulns(self):
         self.assertEquals("Medium", item.impact)
         self.assertEquals("CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H", item.cvssv3)
         self.assertEquals(datetime.date(2021, 2, 12), item.date.date())
-        self.assertEquals("XRAY-97724", item.vuln_id_from_tool)
+        self.assertEquals("XRAY-97724", item.unique_id_from_tool)
 
         # rpm
-        item = [i for i in findings if i.title[:11] == "XRAY-106044"][0]
+        item = [i for i in findings if i.title[:11] == "XRAY-106044"][-1]
         self.assertEquals("CVE-2019-19645", item.cve)
         self.assertEquals("XRAY-106044 - CVE-2019-19645 sqlite: infinite", item.title[:45])
         self.assertEquals("TABLE statements", item.title[-16:])
         self.assertEquals("Medium", item.severity)
         self.assertEquals("alter.c in SQLite", item.description[:17])
         self.assertEquals("TABLE statements.\n\nRed Hat Severity: Moderate", item.description[-45:])
         self.assertIsNone(item.mitigation)
-        self.assertEquals("rpm://7:sqlite:0", item.component_name)
+        self.assertEquals("7:sqlite:0", item.component_name)
+        self.assertEquals('"packagetype_rpm"', item.tags)
```

Comment:
there is double quote `"` in the data. is it something we want?

Reply:
For the tags? This was the syntax I found somewhere for asserting equal for lists (or arrays? whatever tags are). If more tags are added then the test can be edited to e.g.
`self.assertEquals('"packagetype_rpm", "someOtherTag"', item.tags)`

The actual data does not have double quotes in it. From parser.py:
`tags = ["packagetype_" + vulnerability['package_type']]`

Happy to change the tests if there's a clearer way to write this that you know of?

Reply:
This form is good:
```suggestion
        self.assertIn('packagetype_rpm', item.tags)
```